OpenImis.RestApi/Startup.cs

```diff
@@ -32,10 +30,13 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            // Add the DbContext 
-            //services.AddDbContext<IMISContext>(options => options.UseSqlServer(Configuration.GetConnectionString("IMISDatabase")));
+            var configImisModules = Configuration.GetSection("ImisModules").Get<List<ConfigImisModules>>();
+            int lastApiVersion = int.Parse(configImisModules[configImisModules.Count - 1].Version);
```

Comment:
It doesn't means that the latest Module is configured at the end. We can add the new module configuration at the beginning.

Reply:
We can download the maximum value of the API in the file:
`var configImisModules = Configuration.GetSection("ImisModules").Get<List<ConfigImisModules>>();
int lastApiVersion = configImisModules.Max(c => int.Parse(c.Version));`